terms/characteristic.md

```diff
@@ -0,0 +1,7 @@
+# Characteristic
+
+The characteristic of a finite field $F$ is the smallest positive number $n$ that:
+$1 + 1 + ... + 1 = 1 * n = 0$
+
+For example, if the field has characteristic 2, it means that:
+$1 + 1 = 0$
```

Comment:
for the prime field, the characteristic is that prime number?

Reply:
Yes it is.